lms/product/canvas/_plugin/course_copy.py

```diff
@@ -14,7 +14,7 @@ def is_file_in_course(self, course_id, file_id):
             # The Canvas API returns file IDs as ints but the file_id param
             # that this method receives (from our proxy API) is a string.
             # Convert ints to strings so that we can compare them.
-            if str(file["id"]) == file_id:
```

Comment:
Following the pattern we have in other LMSes:

- `id`: our internal ID, with a `urlschema://` format.
- `lmd_id`: raw ID of the file in the LMS.